### PR TITLE
only emails of the 'not submitted' students shown #2987

### PR DESCRIPTION
**Pull request by imouto1994**
Fix the bug and remove the duplication in showing the response rate in different types of sort in results page

### CODE REVIEW

File: src/main/java/teammates/logic/core/FeedbackSessionsLogic.java [hunk 1]
```diff
@@ -1807,6 +1807,11 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
 
                 }
             }
+            if (questionNumber == -1) {
```
Comment (damithc) [inline src/main/java/teammates/logic/core/FeedbackSessionsLogic.java]: Magic number. Define as constant?

File: src/test/resources/pages/instructorFeedbackResultsAjaxByGQR.html [hunk 2]
```diff
@@ -1535,91 +1502,42 @@
...
-                   </td>
-                   <td>
-                      3
-                   </td>
-                   <td>
-                      3
-                   </td>
-                   <td>
-                      -
-                   </td>
```
Comment (damithc) [inline src/test/resources/pages/instructorFeedbackResultsAjaxByGQR.html]: Next time, try to use SourceTree to unstage unrelated hunks before committing.

**Comment by damithc** [comment]
Look through the unrelated changes in html files to ensure they are indeed harmless changes in layout

File: src/main/java/teammates/logic/core/FeedbackSessionsLogic.java (hunk 1 above)
Comment (imouto1994) [inline src/main/java/teammates/logic/core/FeedbackSessionsLogic.java]: Fixed :)

**Comment by imouto1994** [comment]
I have checked and confirmed that changes will only affect the new jsp `instructorFeedbackResultBottom.jsp` :)

**Pull request merged**